lib/cocoapods-packager/builder.rb

```diff
@@ -89,6 +90,7 @@ def build_with_mangling(platform)
 
     def compile(platform)
       defines = "GCC_PREPROCESSOR_DEFINITIONS='PodsDummy_Pods_#{@spec.name}=PodsDummy_PodPackage_#{@spec.name}'"
+      defines += " #{@spec.consumer(platform).compiler_flags.join(' ')}"
```

Comment:
this could just be `defines << @spec.consumer(platform).compiler_flags.join(' ')`